recipes/seqchromloader/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "seqchromloader" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 91efbfad647c3abc3a64880357be3f3b9de5363628b9d6adcf2b6197dccfb647
+
+build:
+  number: 0
+  noarch: generic
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7,<3.10
```

Comment:
If you look at the Azure tests you can see your package is only built for python 3.9. You can build it for python 3.7, 3.8 and 3.9 with these changes:

```
build:
  skip: True  # [py<37 or py>39]

requirements:
  host:
    - python
  run:
    - python
```

by not specifying the python version in the requirements, all versions are build. Using the build skip, you then exclude the versions that don't work for you (with the comment).

Reply:
Turns out this one does not work together with `noarch` on bioconda. [Relevant docs](https://bioconda.github.io/contributor/linting.html#noarch-or-not-noarch).